### PR TITLE
Fix Bundles

### LINKED ISSUE #1609: many bundles are not uploaded
note modification date

note bundle_.zip file

image

### PR DESCRIPTION
**Pull request by chuckwagoncomputing**
This fixes #1609

### CODE REVIEW

File: misc/jenkins/compile_other_versions/prepare_bundle.sh [hunk 1]
```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 
 SCRIPT_NAME="prepare_bundle.sh"
-echo "Entering $SCRIPT_NAME with $BUNDLE_NAME"
+echo "Entering $SCRIPT_NAME with ${1}"
```
Comment (rusefi) [inline misc/jenkins/compile_other_versions/prepare_bundle.sh:4]: shall we 
``BUNDLE_NAME=${1}``
on top?

Comment (chuckwagoncomputing) [reply misc/jenkins/compile_other_versions/prepare_bundle.sh]: Good idea.

File: misc/jenkins/compile_other_versions/prepare_bundle.sh [hunk 2]
```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 
+BUNDLE_NAME="$BUNDLE_NAME"
```
Comment (rusefi) [inline misc/jenkins/compile_other_versions/prepare_bundle.sh:3]: BUNDLE_NAME="$BUNDLE_NAME"?

Comment (chuckwagoncomputing) [reply misc/jenkins/compile_other_versions/prepare_bundle.sh]: Search and replace got me. I even thought of that as I was doing it but somehow still got it wrong.

**Pull request merged**